docs/pages/buttons.vue

```diff
@@ -103,6 +108,57 @@
 
     </DocsPageSection>
 
+    <DocsPageSection title="Icon buttons" anchor="icon_buttons">
+      <p>
+        Use icon buttons for editors, or in situations where using text buttons would hinder the user experience.
+      </p>
+      <p>
+        Default to using flat secondary style for icon buttons.
+      </p>
+      <p>
+        Be sure to consider internationalization, translation, and cultural meaning in the use of icon buttons.
+      </p>
+      <p>
+        Always include a tooltip with the name of the action for icon buttons.
+      </p>
+      <!-- TODO: implement icon button that shows an example tooltip label on hover. See Google doc for example -->
+      <p>
+        <b>TODO: INSERT ICON BUTTON EXAMPLE HERE</b>
+      </p>
+
+    </DocsPageSection>
+
+    <DocsPageSection title="Dropdowns" anchor="dropdowns">
+      <p>
+        There are variations of buttons which can open a dropdown menu:
+      </p>
+      <!-- TODO: implement 3 examples: primary button, primary flat button, and secondary icon button that function as dropdowns. See Google doc for example -->
+      <p>
+        <b>TODO: INSERT DROPDOWN BUTTON EXAMPLES HERE</b>
+      </p>
+
+    </DocsPageSection>
+
+    <DocsPageSection title="Visual specs" anchor="specs" />
+    <ul>
+      <li>Border radius: 2px</li>
+      <li>Elevation: 4dp</li>
+      <li>Font size: 14px</li>
+      <li>Icon size: 14px</li>
+      <li>Text button padding: 8px 16px</li>
+      <li>Icon button padding: 8px</li>
+    </ul>
+    <p>
+      Don't:
+    </p>
+    <ul>
+      <li>Use outlines for buttons</li>
+      <li>Use full-width buttons</li>
+      <li>Set arbitrary fixed widths</li>
+      <li>Change the border radius</li>
+      <li>Change the drop-shadow</li>
+    </ul>
+
```

Comment:
Might be a neat spot to use a `DocsDoNot` component @indirectlylit

Reply:
Oooh yes I forgot about that - I can add this in